stix_shifter_modules/secretserver/stix_translation/json/event_from_stix_map.json

```diff
@@ -13,29 +13,31 @@
       ]
     }
   },
-
-  "user-account": {
+   "user-account": {
     "fields": {
-      "user_id": [
-        "UserId"
-      ]
+      "user_name": "username"
     }
   },
+  "email-addr": {
+        "fields": {
+            "value": "EmailAddress"
+        }
+    },
   "x-secret": {
     "fields": {
       "secret_name": ["SecretName"],
-      "user_name": ["username"],
+      "username": ["Unique_Identtification"],
```

Comment:
Couldn't this go in `user-account:display_name` instead of this custom object.

Reply:
We have done our db parser changes with the above mapping. If it is not mandatory this time let it be like that.

Reply:
How is Unique_Identification different from the username field? This can likely remain where it is if they are different.